packages/data-explorer-ui/src/components/Export/components/ExportToTerra/components/ExportToTerraNotStarted/exportToTerraNotStarted.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ElementType } from "react";
```

Comment:
In the future, we could probably re-use this new component instead of https://github.com/clevercanary/data-explorer/blob/f4c8d8e5e24e8cc895d5928c884fbd759fbbac70/packages/data-explorer-ui/src/components/Export/components/ExportToTerraNotStarted